README.md

```diff
@@ -69,7 +69,46 @@ docker build -t pure-fa-ome:$VERSION .
 
 **Authentication**
 
-Authentication is used by the exporter as the mechanism to cross authenticate to the scraped appliance, therefore for each array it is required to provide the REST API token for an account that has a 'readonly' role. The api-token must be provided in the http request using the HTTP Authorization header of type 'Bearer'. This is achieved by specifying the api-token value as the authorization parameter of the specific job in the Prometheus configuration file.
+Authentication is used by the exporter as the mechanism to cross authenticate to the scraped appliance, therefore for each array it is required to provide the REST API token for an account that has a 'readonly' role. The api-token can be provided in two ways
+
+- using the HTTP Authorization header of type 'Bearer', or
+- via a configuration map in a specific configuration file.
+
+The first option requires to specify the api-token value as the authorization parameter of the specific job in the Prometheus configuration file.
+The second option allows to provide the FlashArray/api-token key-pair map for a list of arrays in a simple YAML configuration file that is passed as paramether to the exporter. This makes possible to write more concise Prometheus configuration files and also to configure other scrapers that cannot use the HTTP authentication header.
+
+### Usage
+
+```shell
+
+usage: pure-fa-om-exporter [-h|--help] [-a|--address "<value>"] [-p|--port
+                           <integer>] [-d|--debug] [-t|--tokens <file>]
+
+                           Pure Storage FA OpenMetrics exporter
+
+Arguments:
+
+  -h  --help     Print help information
+  -a  --address  IP address for this exporter to bind to. Default: 0.0.0.0
+  -p  --port     Port for this exporter to listen. Default: 9491
+  -d  --debug    Enable debug. Default: false
+  -t  --tokens   API token(s) map file
+```
+
+The array token configuration file must have the following synthax:
```

Comment:
spelling correcting 

The array token configuration file must have the following syntax:

Reply:
```suggestion
The array token configuration file must have the following syntax:
```